### ISSUE TITLE
Finalize mechanism to distribute localized assemblies via dotnet pack

### ISSUE THREAD

**Issue by rohit21agrawal** [NuGet/Home#5186]

**Comment by kzu** [comment]
This is already supported in NuGetizer (has mostly always been, rather?)

**Comment by rohit21agrawal** [comment]
Does nugetizer add all localized assemblies to the same nupkg? Or creates nupkg per culture?

**Comment by kzu** [comment]
Adds the satellite resources to the main package

**Comment by clairernovotny** [comment]
In Humanizer, we need to create one package per culture and a metapackage that references all cultures. Would love to use pack targets for this but am currently blocked https://github.com/Humanizr/Humanizer/tree/dev/NuSpecs

**Comment by rohit21agrawal** [comment]
@onovotny pack targets already add satellite assemblies (if they are part of SatelliteDllsProjectOutputGroup) to the nupkg in the culture folder inside lib. Am i missing something that you require?

**Comment by rohit21agrawal** [comment]
@onovotny or is your expectation that pack should create an individual nupkg for each culture and a metapackage that references each of them?

**Comment by clairernovotny** [comment]
Yes....we require creating a language specific nupkg per satellite resource. Not including them in the main package.

**Comment by rohit21agrawal** [comment]
@onovotny can i ask why are you against putting all satellite resources in the main package?

**Comment by clairernovotny** [comment]
The expectation is that we should have that option. We are doing this because some users explicitly don't want all of the languages "polluting their output". I think it's silly, but it's what the users want. By using the satellite packages, people can choose the languages they want. Or they can use the main metapackage and get them all.
